cmd/client/client.go

```diff
@@ -36,7 +36,50 @@ import (
 
 // Config defines the parameters to run a tunnel client.
 type Config struct {
-	TunnelAddress, DialAddress, CertFile, Target, TargetType string
+	TunnelAddress, DialAddress, ListenAddress, CertFile, Target, TargetType string
+	Bridge                                                                  bool
```

Comment:
I thought we removed Bridge from all configs?

Reply:
probably better remove that flag. It is just to enable creating a new session from the client side. 

updated the request.